src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -2,93 +2,20 @@
 
   <PropertyGroup>
     <TargetFrameworks>net6.0;net472</TargetFrameworks>
-    <IsPackable>true</IsPackable>
-    <IsShippingPackage>true</IsShippingPackage>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <PackageType>MSBuildSdk</PackageType>
-    <VersionPrefix>1.0.0</VersionPrefix>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <NoWarn>$(NoWarn);NU5128</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\Tasks\Common\**\*.cs" LinkBase="Common" />
+    <!-- ExcludeAssets="Runtime" is being set to avoid adding package references and dependencies of project and package references. -->
```

Comment:
I think the object model packages should carry their own copies of things as if they are referenced as packages they should work fine, but on the MSBuild task project we should exclude the assets.

This goes against my comment of letting NuGet flow transitively, but now that I give it a second thought it makes more sense to just exclude things on the MSBuild task and include the references we want to carry as the task, and leave these object model packages as regular packages and not making them special for the first entry point we added.